packages/expo-audio/src/AudioModule.web.ts

```diff
@@ -237,14 +236,16 @@ export class AudioRecorderWeb
 
   id = nextId();
   _options: Partial<RecordingOptions>;
-  _mediaRecorder?: MediaRecorder;
+  _mediaRecorder: MediaRecorder | null = null;
```

Comment:
To be honest I'm more in favor of using `private` properties from TypeScript than using a `_` prefix.

Reply:
I assumed when it was fine [here](https://github.com/expo/expo/blob/816e117ee0cca947b7957e3e3eb240b1faaa8dc5/packages/expo-video/src/VideoPlayer.web.tsx#L40-L54) I'd keep it the same 😅 It makes it easier to add getters with the same name except for the prefix instead of renaming the backing property. I can change it though